### DISCUSSION TITLE
Remove Trigger Key

### DISCUSSION THREAD

**Discussion by LisonFan** [mrkai77/Loop#353] [Q&A]
I just need the middle click of the mouse scroll wheel to trigger, I don't need keyboard shortcuts to trigger.

![image](https://github.com/MrKai77/Loop/assets/9424775/d848eec7-e7d2-4989-8d4b-a90145af0d6e)

**Comment by mrkai77** [comment]
That is currently impossible, but as a workaround, I would recommend simply setting it to keys that you would never press, so that only the middle click button functionally works :)
